src/constraints.jl

```diff
@@ -51,6 +51,32 @@ end
 
 Base.broadcastable(con_ref::ConstraintRef) = Ref(con_ref)
 
+"""
+    dual_start_value(con_ref::ConstraintRef)
+
+Return the dual start value (MOI attribute `ConstraintDualStart`) of the constraint `con_ref`. 
+
+Note: If no dual start value has been set, `dual_start_value` will return `Nothing`
+
+See also [`set_dual_start_value`](@ref).
+"""
+function dual_start_value(con_ref::ConstraintRef)::Union{Nothing, Float64}
```

Comment:
What about vector constraints?
The current implementations is only for ScalarShape?